src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -19,52 +19,6 @@
 /* data cache line alignment */
 #define PLATFORM_DCACHE_ALIGN	DCACHE_LINE_SIZE
 
-#define SRAM_BANK_SIZE			(64 * 1024)
```

Comment:
This definition is in cavs25/adsp_memory.h in Zephyr, not needed here in SOF codebase. All below stuff is unused stuff that can be removed.